python/lsst/meas/deblender/sourceDeblendTask.py

```diff
@@ -353,14 +353,14 @@ def deblend(self, exposure, srcs, psf):
             if self.isLargeFootprint(fp):
                 src.set(self.tooBigKey, True)
                 self.skipParent(src, mi.getMask())
-                self.log.warning('Parent %i: skipping large footprint (area: %i)',
-                                 int(src.getId()), int(fp.getArea()))
+                self.log.debug('Parent %i: skipping large footprint (area: %i)',
```

Comment:
I think I'd lean on this one being `info` level, since this is a bit more of a "something is wrong" skip.

Reply:
Something isn't wrong -- per the [docs for isLargeFootprint](https://pipelines.lsst.io/v/daily/py-api/lsst.meas.deblender.SourceDeblendTask.html#lsst.meas.deblender.SourceDeblendTask.isLargeFootprint), it's supposed to flag things like satellite streaks. (And if it _were_ a problem, [INFO wouldn't be an appropriate level](https://docs.python.org/3.11/library/logging.html#levels).)